sdk/core/test/integration/otp/index.test.ts

```diff
@@ -80,7 +80,7 @@ describe('CommonNetworkMember [OTP]', () => {
     commonNetworkMember = new CommonNetworkMember(password, encryptedSeed, options)
 
     await commonNetworkMember.storeEncryptedSeed('', '', accessToken)
-    await commonNetworkMember.signOut()
+    await commonNetworkMember.signOut(options)
```

Comment:
Unfortunately it is a legacy code, `commonNetworkMember.signOut` expects `SdkOptions` as a parameter and does not use options that were passed to the constructor before. The only relevant change here is that now options are required (because we no longer use `dev` as a default environment).

Also see https://lemmatree.slack.com/archives/C025J7PEG80/p1624032996226700?thread_ts=1624032991.226600&cid=C025J7PEG80

Reply:
I fully agree that it should use class options instead. I wish someone told us why was it made that way. Apparently it was that way from the beginning: https://github.com/affinityproject/affinidi-core-sdk/blob/3a28e957b061dcdaf1e063d7168e5cd0df1e3eec/sdk/core/src/CommonNetworkMember.ts#L641